### PR TITLE
Dls 8445 create packaging site details

### PR DESCRIPTION
**Pull request by ginnyamazed**
Screenshot 2023-07-27 at 13 48 45
Screenshot 2023-07-27 at 13 49 05
Screenshot 2023-07-27 at 13 50 30

### CODE REVIEW

**Comment by ginnyamazed** [comment]
UI page addition, https://github.com/hmrc/soft-drinks-industry-levy-ui-tests/pull/143#issue-1826426587

**Comment by RaminEsfandiari** [comment]
Not part of this change but I noticed you had a duplicate route in cancel registration. Could you remove that either as part of this PR?
image

File: app/navigation/NavigatorForCorrectReturn.scala [hunk 1]
```diff
@@ -89,8 +88,10 @@ class NavigatorForCorrectReturn @Inject()() extends Navigator {
   }
 
   override val normalRoutes: Page => UserAnswers => Call = {
+
     case RemovePackagingSiteConfirmPage => _ => defaultCall
     case SmallProducerDetailsPage => _ => defaultCall
+    case PackagingSiteDetailsPage => _ => defaultCall
```
Comment (RaminEsfandiari) [inline app/navigation/NavigatorForCorrectReturn.scala:94]: We might need to think about the navigation as three different variation journeys end up on this call and they may have different navigation requirements but we'll cross that bridge when we come to it.

File: app/views/changeActivity/PackagingSiteDetailsView.scala.html
Comment (RaminEsfandiari) [inline app/views/changeActivity/PackagingSiteDetailsView.scala.html:1]: Thanks for fixing

File: app/views/correctReturn/PackagingSiteDetailsView.scala.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+        }
+
+        @if(packagingSites.size == 1) {
+            <h1 class="govuk-heading-m">@messages("correctReturn.packagingSiteDetails.heading1")</h1>
+            } else {
+                <h1 class="govuk-heading-m">@messages("correctReturn.packagingSiteDetails.heading", packagingSites.size)</h1>
+            }
```
Comment (RaminEsfandiari) [inline app/views/correctReturn/PackagingSiteDetailsView.scala.html:43]: Another way to achieve this is to send to parameters to your message, first being the size second a conditional pluraliser like "s"
See link for an example:
https://github.com/hmrc/soft-drinks-industry-levy-registration-frontend/blob/main/app/views/WarehouseDetailsView.scala.html#L28

**Review by RaminEsfandiari — CHANGES_REQUESTED** [review]
Looks good. Just the removal of duplicate route from previous changes and happy to approve

**Review by RaminEsfandiari — APPROVED** [review]

**Pull request merged**